src/reducers/paths.reducer.js

```diff
@@ -42,12 +45,29 @@ export default (state: State = initialState, action: Action) => {
 //
 //
 // Helpers
-export const getDefaultParentPath = () =>
+export const getDefaultParentPath = () => {
   // Noticing some weird quirks when I try to use a dev project on the compiled
   // "production" app, so separating their home paths should help.
-  process.env.NODE_ENV === 'development'
-    ? `${os.homedir()}/guppy-projects-dev`
-    : `${os.homedir()}/guppy-projects`;
+
+  // For Windows Support
+  // Documents folder is much better place for project folders (Most programs use it as a default save location)
+  // Since there is a chance of being moved or users language might be differet we are reading the value from Registery
+  // There might be a better solution but this seems ok so far
+  const winDocumentsRegRecord = childProcess.execSync(
+    'REG QUERY "HKCU\\Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\User Shell Folders" /v Personal',
+    {
+      encoding: 'utf8',
+    }
+  );
+  // prettier-ignore
+  const winDocumentsPath = winDocumentsRegRecord.split('   ')[winDocumentsRegRecord.split('   ').length - 1].replace('%USERPROFILE%\\', '').replace(/\s/g, '');
```

Comment:
Do you happen to know how consistent windows registry formats are between versions of windows? Would this work from Windows 7 through to Windows 10, for example?

IIRC registry stuff tends not to change much, so I believe it's safe, but I haven't used Windows in over a decade 😅so my info is a bit rusty

Reply:
I changed it a little bit. Initially split was creating bunch of empty values, that is why I replaced bunch of spaces. With single space split it should work in prior windows versions too. I'm going to try it on Win7 soon.